.circleci/config.yml

```diff
@@ -198,7 +198,6 @@ workflows:
             - prep-build-test-flask
             - get-changed-files-with-git-diff
       - test-e2e-firefox-flask:
-          <<: *main_master_rc_only
```

Comment:
thanks for checking this! I've seen that the [job is green](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/119017/workflows/4bb6f0a9-cede-4997-9e66-e5e903cb1b12/jobs/4444956) but the Send SOL spec is flaky.

I'd rather investigate the source of flakiness and fix it before merging the PR, otherwise we'll get the problem in `main`.



![Screenshot from 2025-01-14 15-48-00](https://github.com/user-attachments/assets/19687feb-2502-4b14-9ad4-893ac2480abd)


Let me know if you have trouble with that :pray: happy to help if needed
Once fixed and verified the job in ci, we need to remember to add the filter back before merging the PR :+1: